app/index.js

```diff
@@ -784,6 +784,11 @@ class GlobalBindings {
     this.requestMove = (user, channel) => {
       if (this.connected()) {
         user.model.setChannel(channel.model)
+        // reflect this change in URL
+        let currentUrl = url.parse(document.location.href, true)
+        currentUrl.query.channelName = channel.name()
```

Comment:
Does this work properly with nested channels?

Reply:
Good point, I haven't tested it and that probably does not work. Let me fix that.